Python/tests/test_runner_number.py

```diff
@@ -27,10 +27,12 @@ def test_number_recognizer(culture, model, options,
 
         assert_verbose(actual.type_name, expected['TypeName'], spec_info)
         assert_verbose(actual.text, expected['Text'], spec_info)
-        assert_verbose(actual.start, expected['Start'], spec_info)
-        assert_verbose(actual.end, expected['End'], spec_info)
         assert_verbose(actual.resolution['value'], expected['Resolution']['value'], spec_info)
 
+        if 'Start' in expected and 'End' in expected:
```

Comment:
Why the need for these? Are 'start' and 'end' missing onlyin Italian specs? If so, we should fix the specs.

Reply:
> 

Yes there are many tests missing the `Start` or `End` values. I noticed this in Italian and German.

Reply:
Could you write a script to add the missing start/end in the specs instead? ;-)

Reply:
I'll take a look into this